packages/zosuss/src/Shell.ts

```diff
@@ -45,6 +45,9 @@ export class Shell {
                     let isUserCommand = false;
                     let rc: number;
 
+                    //isolate the command
+                    const cmd = command.split(' ')[0];
```

Comment:
Sorry I realized one of the tests I tried on the previous PR isn't working anymore when my SSH shell is `sh`:
> 2. Command that contains itself in output: `cat test.sh` where the contents of "test.sh" are:
>     ```sh
>     cat pre-test.sh
>     cat test.sh
>     cat post-test.sh
>     ```

I think instead of just grabbing the first word of the command (in this case "cat"), maybe we want to grab up to the first 80 characters (before it would wrap)?
`const cmd = command.slice(0, 80)`

Reply:
doesn't work for "echo 'kjsdf kahdfkhjsd kahsdfkjas aksdjhfkajsdhfkhjasdf asdkfjhajksdhfkahjsdf asdkjfhajksd'" in bash

Reply:
After discussing in person we concluded that first line of the command seems to get wrapped at 75 characters.

This is probably because the `ssh2` package has a default of 80 columns for terminal width.
We subtract 5 from that because of `\r\n$ ` at the beginning plus a line ending character at the end.